pkgs/top-level/all-packages.nix

```
@@ -1124,6 +1124,8 @@ in
 
   passExtensions = recurseIntoAttrs pass.extensions;
 
+  passwordsafe = callPackage ../applications/misc/passwordsafe { };
```

Probably should be gnome-passwordsafe since that is what you called it. Looks like there is something else called passwordsafe too: https://repology.org/projects/?search=passwordsafe
```suggestion
  gnome-passwordsafe = callPackage ../applications/misc/gnome-passwordsafe { };
```

---

Okay, I was wondering also if it should not go in gnome3.passwordsafe? I am not sure what goes into deciding that. Should I ping a gnome maintainer?

---

Hey, a gnome maintainer found this :grin: In the case of any gtk application pinging a gnome maintainer is necessary because gtk packaging can be confusing. We're on the fence with there being `gnome3.*` at all, so I'd `gnome-passwordsafe` would be the best attribute for nixpkgs cc @jtojnar

---

On another topic, the organization of `applications/misc` isn't optimal. I believe this would be a good chance to add a new directory `applications/password-managers`. What do you think?

---

I found this here, but maybe tools isn't a good directory either? pkgs/tools/security/1password-gui/

---

also: `pkgs/tools/security/gnome-keysign`